### DISCUSSION TITLE
Point blank new for catchon

### DISCUSSION THREAD

**Discussion by CrazyDibber** [3kh0/ext-remover#451] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi, I was on catchon(spyware for school) and I found this: 
you think there is a way to implement point blank for this?

<img width="2240" alt="Screenshot 2023-01-27 at 5 12 30 PM" src="https://user-images.githubusercontent.com/114046772/215233452-37621dbb-511d-4240-8942-e4da53d9f9c9.png">

**Comment by Aka-but-nice** [comment]
> Hi, I was on catchon(spyware for school) and I found this: you think there is a way to implement point blank for this?
> 
> <img alt="Screenshot 2023-01-27 at 5 12 30 PM" width="2000" src="https://user-images.githubusercontent.com/114046772/215233452-37621dbb-511d-4240-8942-e4da53d9f9c9.png">
🫣

**Comment by Aka-but-nice** [comment, marked as answer]
No, it doesn't even have a link, or anything on the page. Sorry, but this won't be possible, it might if you show me the block page, if the extension block page is a extension link, instead of a web link, this may be possible, with either encoding the URI Component into it, or using html directly

**Reply by Crazygird** [reply]
ok
